src/views/Gantt.vue

```diff
@@ -0,0 +1,267 @@
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div class="c-gantt">
+    <v-container
+      fluid
+      class="pa-2"
+    >
+      <!-- Filters -->
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          md="4"
+          class="pr-md-2 mb-2 mb-md-0"
+        >
+          <v-autocomplete
+            multiple
+            chips
+            closable-chips
+            clearable
+            placeholder="Search"
+            :items="callback.uniqueTasks"
+            v-model="jobsFilter.name"
+            label="Select tasks"
+            ref="selectTasks"
+            @update:search="updateSelectionOptions"
+          >
+            <template
+              v-slot:prepend-item
+              v-if="this.showSelectAll"
+            >
+              <v-list-item
+                ripple
+                @click="selectSearchResults"
+              >
+                Select all search results
+              </v-list-item>
+              <v-list-item
+                ripple
+                @click="deselectSearchResults"
+              >
+                Remove all search results
+              </v-list-item>
+            <v-divider class="mt-2"></v-divider>
+            </template>
+          </v-autocomplete>
+        </v-col>
+        <v-col
+          cols="12"
+          md="4"
+          class="mb-2 mb-md-0"
+        >
+          <v-select
+            id="c-gantt-filter-job-timings"
+            :items="timingOptions"
+            prefix="Displaying:"
+            v-model="jobsFilter.timingOption"
+          />
+        </v-col>
+        <v-col
+          cols="12"
+          md="4"
+          class="pl-md-2 mb-2 mb-md-0"
+        >
+          <v-select
+            id="c-gantt-filter-job-platforms"
+            :items="platformOptions"
+            prefix="Platform:"
+            v-model="jobsFilter.platformOption"
+          />
+        </v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          md="4"
+          class="pr-md-2 mb-2 mb-md-0"
+        >
+          <v-select
+            id="c-gantt-tasks-per-page"
+            :items="$options.taskChoices"
+            prefix="Tasks Per Page"
+            v-model="tasksPerPage"/>
+        </v-col>
+      </v-row>
+      <div
+        id="gantt-toolbar"
+        class="d-flex align-center flex-wrap my-2 col-gap-2 row-gap-4">
+
+        <!-- Toolbar -->
+        <v-defaults-provider>
+
+          <!-- Box plot sort input teleports here -->
```

Comment:
Not quite sure what this means?

Reply:
Remnants of old code that has now been removed.

Reply:
The skeleton loader has been added. I don't have a workflow big enough to cause long enough loads.
Can you test it quickly for me

Reply:
The skeleton loader needs to be toggled according to the load state. Here's an example from the log view:

https://github.com/cylc/cylc-ui/blob/e846f8646582c1f00906af358bf646ceef56670a/src/views/Log.vue#L116-L120

The skeleton loader will only display here if "results.connected" is falsey.

I'll send you a link to my workflow DB.

Reply:
Ronnie was kind enough to implement skeleton loaders properly for me